rero_ils/modules/deduplications/api.py

```
@@ -38,7 +38,7 @@
 class Deduplication:
     """Document deduplication class."""
 
-    def __init__(self, es_hosts=[]) -> None:
+    def __init__(self, es_hosts=[]):
```

Ruff (0.15.7)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)